src/util/entropyCreateStateFromJsons.ts

```diff
@@ -300,12 +347,13 @@ function _frame(start:number, end:number, phase: number, genomeLength:number, po
  * refers to this being a stacking problem.) The stack position starts at 1.
  * Returns the maximum position observed.
  */
-function calculateStackPosition(genes: Gene[], strand: (Strand|null) = null):number {
+function calculateStackPosition(
+  genes: Gene[],
+  strand: Strand,
+): number {
   /* List of CDSs, sorted by their earliest occurrence in the genome (for any segment) */
   let cdss = genes.reduce((acc: CDS[], gene) => [...acc, ...gene.cds], []);
```

Comment:
This series of statements could be one stacked `.reduce().filter().sort()` operation, which would then let `cdss` be a `const` declaration instead of a `let`.

I don't think it would be that much harder to read, that way.

Reply:
Sure, but out of scope of this PR. I'll open another PR for that change.

Reply:
Actually it seems fine to bundle with cad5eb63. Added as 28bd3ed9a4b1a03aeee9ddf9da6015d7679880e9.